.gitignore

```diff
@@ -15,10 +15,10 @@ pip-selfcheck.json
 
 # coverage
 .coverage
+htmlcov/
 
-# emacs
+# editors
 pml/#*
```

Comment:
I don't think we need the pml prefix here.  Any file beginning with # should be ignored.